src/lib/color-picker/color-picker-collection.component.html

```diff
@@ -10,11 +10,9 @@ <h3>{{ label }}</h3>
                 [mccColorPickerOption]="color" (click)="setColor(color)">
 
           <ng-container *ngIf="!disableSelectedIcon">
-            <mat-icon *ngIf="selectedColor === color; else emptyIcon"
-                      [svgIcon]="selectedSvgIcon">{{ selectedIcon }}</mat-icon>
-            <ng-template #emptyIcon>
-              <mat-icon></mat-icon>
-            </ng-template>
+            <mat-icon *ngIf="selectedColor !== color"></mat-icon>
+            <mat-icon *ngIf="selectedColor === color && !selectedSvgIcon" class="mat-font-icon">{{ selectedIcon }}</mat-icon>
+            <mat-icon *ngIf="selectedColor === color && selectedSvgIcon"  class="mat-svg-icon" [svgIcon]="selectedSvgIcon"></mat-icon>
```

Comment:
All 3 if's will be always checked.
It's better to isolate the second and third if inside an `ng-template` tag only validating if `selectedSvgIcon` is set.

Reply:
i changed it